### DISCUSSION TITLE
Multiple Wallets per single user

### DISCUSSION THREAD

**Discussion by mrcfx** [Web3Auth/Web3Auth#628] [Help]
How can I correctly implement multiple wallets per individual user?

Ex. user a is assigned 10 ETH wallets and 5 Solana wallets

**Comment by himanshuchawla009** [comment]
Web3auth returns a app scoped private key which is 32 byte key and you can use this key as seed in bip39 lib to derive a seed phrase and use that seed phrase to derive n number of deterministic wallet keys using bip44.
